.github/workflows/build.yml

```diff
@@ -1,51 +1,69 @@
 name: Build
+
 on:
   push:
     branches:
       - '*'
   pull_request:
     branches:
       - master
+
 jobs:
   build:
     strategy:
       matrix:
-        platform: [ubuntu-latest]
+        platform: [ubuntu-latest, windows-latest]
         otp-version: [21.3, 22.2]
+
     runs-on: ${{ matrix.platform }}
+
     container:
       image: erlang:${{ matrix.otp-version }}
+
     steps:
+    - name: Install Erlang
+      runs-on: windows-latest
+      run: choco install erlang
```

Comment:
Is the idea (in the final version) to re-use the versions of Erlang from the `otp-version` field for Windows as well?

Reply:
It is, I just need to get anything to run right now...

Reply:
Sure. And thanks for the help!